Sources/ApolloAPI/GraphQLOperation.swift

```diff
@@ -70,6 +71,10 @@ public extension GraphQLOperation {
     return nil
   }
 
+  static var hasDeferredFragments: Bool {
+    false
```

Comment:
This will get set for each operation that has deferred fragments in it as part of the work for https://github.com/apollographql/apollo-ios/issues/3114.